packages/apps/transfer/.eslintrc.js

```diff
@@ -1,8 +1,32 @@
+const fs = require('fs');
+
+const prettierOptions = JSON.parse(fs.readFileSync('./.prettierrc', 'utf8'));
```

Comment:
importing prettier this way helps to sync IDEs such as webstorm which has it's own formatted with project standards
cause now both IDE and linter will read prettier config from same config file